### PR TITLE
ci: semantic PR title via conventional commits

### PR DESCRIPTION
**Pull request by scarf005**
## Purpose of change

- enforce [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for PR titles
- remove redundant `## Summary`

## Describe the solution

install https://github.com/apps/semantic-prs to enforce PR title

## Describe alternatives you've considered

pain

## Testing

used them on other projects, they work as intended

## Additional context

Q: Nooo, do i have to edit all those PR titles, i don't know how to do that
A: No worries, i've been doing that manually for months and this app just prevents me from overlooking one

Q: Then can i just write whatever on PR title and you'll fix it for me?
A: It'd be very nice if you could follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), tho.

## Checklist

- [x] reword automated i18n PR title
- [x] remove `## Summary` section
- [x] think up of better category
- [x] update docs

## Additional Context

image

Semantic PR bot also needs to be approved to enforce the PR title consistency.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,32 +12,20 @@ WARNING: If autofix.ci app did the formatting for you, YOU MUST DO EITHER OF THE
...
+feat(content, mods): new item for <mod name>
+
+for more on which category is available, see:
 https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/
 
-If the PR is a port or adaptation of DDA content, please indicate it to be so.
+If the PR is a port or adaptation of DDA content, please indicate it by adding port in PR title, like:
+feat(port): <feature name> from DDA
+
+COMMENTS: Please remove all comments and unrelevent sections from this template before submitting your PR.
```
Comment (Coolthulhu) [inline .github/pull_request_template.md:28]: I'm not sure if I understand this bit.
Also, seems to be a typo: unrelevAnt.

**Comment by Coolthulhu** [comment]
Looks good to me, I'll approve the semantic PR thing for now. But I'd like to know what exactly is the "no comment" section about - I hope it doesn't mean that the submitter will have to manually prune the PR notes.

File: .github/pull_request_template.md (hunk 1 above)
Comment (scarf005) [reply .github/pull_request_template.md:28]: oh, i meant pruning html comments in PR body, as they get included in PR body previews (e.g on notification)

**Review by Coolthulhu — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (olanti-p) [reply .github/pull_request_template.md]: Before #3519, the template used to ask to remove the comments, but turns out ~40% of PRs don't do that anyway (either laziness or just avoiding removing something that may become relevant), so we're stuck with them until GitHub introduces issue forms, but for PRs.

**Pull request merged**